std_msgs/Header header
geometry_msgs/Point left_xyz
geometry_msgs/Point right_xyz